isis/src/mro/apps/hi2isis/main.cpp

```diff
@@ -36,6 +37,228 @@ int hisCount[6];
 int validCount[6];
 int section;
 
+
+// Construct a BLOb to contain the Hirise main line suffix and prefix data
+void SaveHiriseAncillaryData(ProcessImportPds &process, Cube *ocube) {
```

Comment:
Was originally floating in another .cpp file, only used in main so moved in to main.